en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -1,96 +1,81 @@
 # Perform Root Cause Analysis
 
-Performing root cause analysis is crucial in identifying and rectifying the underlying problem for any anomalies detected in a system. Choreo Observability provides helpful insights out-of-the-box that assists in carrying out a root cause analysis for any service deployed on the Choreo platform.
+Choreo provides insightful observability features to drill down to the request level and confirm the root cause for anomalies you detect.
 
-!!! info
-    For explanation, we are using the sample data available in the product.
-    
-## Get started with the sample service
+This guide describes how you can detect anomalies and perform root cause analysis on a service deployed in Choreo. For demonstration, let’s use the observability sample service available in Choreo.
 
-1. Go to **Services** from the left navigation.
-2. Create a new service.
-    1. **If this is your first service**, 
     
-        1. Enter a name for the service and click **Create**. 
-           
-           ![Create a new service](../assets/img/observability/first-service-creation.png){.cInlineImage-full}
-           
-    2. **Else** (if you already have services created), 
-        
-        1. Click **+ Create** to start creating a new service.
-        2. Enter a name for the service and click **Create**.
-           
-           ![Create a new service](../assets/img/observability/service-creation.png){.cInlineImage-full}
-
-3. From the left-hand menu, click **Observe**.
-4. On the right-hand bottom, click the **Sample Service** icon.
-   
-    ![Sample service popup](../assets/img/observability/sample-service-pop-up.png){.cInlineImage-full}
-   
-5. Click **Try Sample Service**.
-
-    ![Try sample service](../assets/img/observability/try-sample-service.png){.cInlineImage-half}
+## Step 1: Get started with the sample service
+Follow this procedure to try out the sample service:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
+2. Go to the **Observability** card and click **Try Now**. This takes you to the Observability page. 
```

Comment:
In "Observability Page", shouldn't "Observability" be bold because it is a page name?

Reply:
+1
Updated.